src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeExceptionHelpers.cs

```diff
@@ -304,374 +332,5 @@ public static bool SafeToPerformRichExceptionSupport
-
-        /// <summary>
-        /// Counter for exception ID assignment
-        /// </summary>
-        private static int s_currentExceptionId;
-
-        /// <summary>
-        /// This method will call the runtime to gather the Exception objects from every exception dispatch in
-        /// progress on the current thread.  It will then serialize them into a new buffer and pass that
-        /// buffer back to the runtime, which will publish it to a place where a global "minidump auxiliary
-        /// provider" will be able to save the buffer's contents into triage dumps.
-        ///
-        /// Thread safety information: The guarantee of this method is that the buffer it produces will have
-        /// complete and correct information for all live exceptions on the current thread (as long as the same exception object
-        /// is not thrown simultaneously on multiple threads). It will do a best-effort attempt to serialize information about exceptions
-        /// already recorded on other threads, but that data can be lost or corrupted. The restrictions are:
-        /// 1. Only exceptions active or recorded on the current thread have their table data modified.
-        /// 2. After updating data in the table, we serialize a snapshot of the table (provided by ConditionalWeakTable.Values),
-        ///    regardless of what other threads might do to the table before or after. However, because of #1, this thread's
-        ///    exception data should stay stable
-        /// 3. There is a dependency on the fact that ConditionalWeakTable's members are all threadsafe and that .Values returns a snapshot
-        /// </summary>
-        public static void GenerateExceptionInformationForDump(Exception currentException, IntPtr exceptionCCWPtr)
-        {
-            LowLevelList<byte[]> serializedExceptions = new LowLevelList<byte[]>();
-
-            // If currentException is null, there's a state corrupting exception in flight and we can't serialize it
-            if (currentException != null)
-            {
-                SerializeExceptionsForDump(currentException, exceptionCCWPtr, serializedExceptions);
-            }
-
-            GenerateErrorReportForDump(serializedExceptions);
-        }
-
-        private static void SerializeExceptionsForDump(Exception currentException, IntPtr exceptionCCWPtr, LowLevelList<byte[]> serializedExceptions)
-        {
-            const uint NoInnerExceptionValue = 0xFFFFFFFF;
-
-            // Approximate upper size limit for the serialized exceptions (but we'll always serialize currentException)
-            // If we hit the limit, because we serialize in arbitrary order, there may be missing InnerExceptions or nested exceptions.
-            const int MaxBufferSize = 20000;
-
-            int nExceptions;
-            RuntimeImports.RhGetExceptionsForCurrentThread(null, out nExceptions);
```

Comment:
You can delete `GetExceptionsForCurrentThread` from everywhere too.

Reply:
I do not think that this was addressed. (Ok to ignore it you think that we may need `GetExceptionsForCurrentThread` at some point in future.)